packages/renderer/src/lib/actions/ContributionActions.svelte

```diff
@@ -98,12 +99,13 @@ async function executeContribution(menu: Menu): Promise<void> {
 </script>
 
 {#each filteredContributions as menu}
-  <ListItemButtonIcon
-    title="{menu.title}"
-    onClick="{() => executeContribution(menu)}"
-    menu="{dropdownMenu}"
-    icon="{getIcon(menu)}"
-    detailed="{detailed}"
-    disabledWhen="{menu.disabled}"
-    contextUI="{globalContext}" />
+  <When expression="{menu.disabled}" contextUI="{globalContext}" let:enabled>
```

Comment:
Isn't the expression the opposite? `!menu.disabled` 
This way, you could use `enabled="{enabled}" in the `ListItemButtonIcon` which is meaningful

Reply:
You cannot use `!menu.disabled` because it is a string.

Here the way we store the expression is `disabled`, which is a negation, I would rather output from the When component `enabled` than `disabled`, which IMO would not make sense.

So at the end I am getting the result of the expression through the enabled properly, which I agree could probably be replaced by a better wording, `value` ? `result`, but the When component does not care what you do with the value, it take an expression and evaluate it.

But I am not outputting the negation of the result of the evaluation of the expression. This is not the responsibility of the When component here.

Reply:
ok, I didn't get that `menu.disabled` was the formulae, and not a boolean. But my concern remains, that we have to use `enabled="{!enabled}"` at the end, which is quite disturbing. 

Could it be possible to let to the user of the When component the choice of the name for the `let:` attribute, so the user would choose the more appriopriate one - here, `let:disabled` for example

Reply:
> Could it be possible to let to the user of the When component the choice of the name for the let: attribute

Trying to do that is probably not something I want to do, as it would definitely be some ugly js code to achieve this specific mechanism.

Moreover it would be very difficult to ensure type safety.

Therefore, I would be against that proposal. But I think it might be interesting to rename `let:enabled` to something different, to remove some potential confusion as you raised.

Reply:
yeah, if Svelte is not providing some simple mechanism, it would be better to have a generic name instead

Reply:
I renamed to `result`